### ISSUE TITLE
Using radar-plots with influxDB

### ISSUE THREAD

**Issue by laimaretto** [snuids/grafana-radar-panel#1]
Hi! I’ve tried your plugin. It’s very very nice … thank you …

Now, I have a couple of questions. In the readme file it's been explained that the plugin in has been tested out with ElasticSearch as data-source (I’m using influxdb).

Let me show you an example of what I'm trying to do: I’m trying to measure the number of messages (LSA) per OSPF Area in a network. To do this, I’ve constructed the following query …

![image](https://user-images.githubusercontent.com/8891825/31731103-55744d5a-b40a-11e7-95c9-1f3c6ee138ac.png)

… and obtained this plot, which is fine and consistent with other plots that I’ve made for the same KPI; so far, so good…

![image](https://user-images.githubusercontent.com/8891825/31731092-50e2d2de-b40a-11e7-917b-cebbddc4f31b.png)

Now, as far as I can understand, the plugin is using time (EPOCH) as the axis for the radar. If you see, because of that, I'm limiting the search to the last 6 samples (hence the hexagon).

What I’d like to achieve is that the ‘tags’ of the influx measurement to be these axis (in my case: ‘0.0.0.0’, ‘0.0.0.2’, ‘0.0.2.4’). So, if that could be possible, instead of an ‘hexagon’, I would have a ‘triangle’, being each corner of it, each of the influx measurement's tags.

Do you think that is possible to achieve with influxdb?

thanks!

lucas

**Comment by snuids** [comment]
Did you try to add another group ? In Elastic, the first aggregation group is used for the diagonals and the second to determine the series names. 

I did not play with Influx DB since a long time but I will try to check that by the end of the week if I have a few hours to spare. Tell me if the dual groups trick does work for you.

**Comment by snuids** [comment]
I committed a new version can you check against it. There is a new option or Influx in the configuration panel.

**Comment by laimaretto** [comment]
Hi! Sorry for the delay. So I've tried your update and it's much better!

![image](https://user-images.githubusercontent.com/8891825/31867393-1ecb8a80-b765-11e7-8dd9-140af1c51a3a.png)

Now the plugin uses tags as axis which is pretty good. In the example, 1Gbps, 10Gbps, etc, are tags, while rxRate and txRate are the fields/KPIs of the influx measurement.

I've noticed, though, that to plot several fields, these must be obtained from different queries ...

![image](https://user-images.githubusercontent.com/8891825/31867402-686937c8-b765-11e7-8c48-feb7591aec96.png)

I believe that the possibility of establishing several fields on the same query, is a plus. Also, being able to define the units of the axis would be nice.

Thanks any way for the update! I'll keep on using it to see if I find anything else!

Thanks again!
